src/query-builder/UpdateQueryBuilder.ts

```diff
@@ -441,6 +441,13 @@ export class UpdateQueryBuilder<Entity> extends QueryBuilder<Entity> implements
                 });
             });
 
+            // It won't throw if trying to trigger update for computational columns such as version/update column
+            // If wanted to update some columns but all of them is not updatable then throw error
+            if (updateColumnAndValues.length <= 0 && Object.keys(valuesSet).length > 0) {
+                // TODO - throw better error
+                throw new UpdateValuesMissingError();
```

Comment:
we already have this throw below. Maybe you just put lines 451-455 into this `if` statement?

Reply:
I can, but don't you wanna throw better error explaining the error?

Reply:
dunno, we can do it next time. I don't think separate error message is required for the case if user wants to update but update contains only `update: false` columns.